src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28078.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue28078 : _IssuesUITest
+{
+	public Issue28078(TestDevice testDevice) : base(testDevice)
+	{
+	}
+	public override string Issue => "Crash occurs when click on flyout menu";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void ShellFlyoutMenuCrash()
+	{
+		App.WaitForElement("button");
+		App.Tap("button");
+		App.WaitForElement("label");
+		App.TapShellFlyoutIcon();
+		VerifyScreenshot();
```

Comment:
Pending snapshot on Windows.
![image](https://github.com/user-attachments/assets/d7944c69-5594-41f4-b759-44d578b8ead6)
Could you commit it?

Reply:
Also pending on Mac.

Reply:
@jsuarezruiz In the last CI trigger, the Mac snapshot was not generated, and only the Windows snapshot was generated. I have committed the Windows snapshot. Could you please retrigger the CI to generate the Mac snapshot?